ramalama/cli.py

```diff
@@ -38,6 +38,12 @@ def init_cli():
     )
     parser.add_argument("--store", default=get_store(), help="store AI Models in the specified directory")
     parser.add_argument("--dryrun", action="store_true", help="show container runtime command without executing it")
+    parser.add_argument(
+        "--runtime",
+        default="llama.cpp",
+        choices=["llama.cpp", "vllm"],
+        help="Specify the runtime to use, valid options are 'llama.cpp' and 'vllm'",
```

Comment:
We are defaulting to lowercase help messages now.

Reply:
Could you also fix:

    parser.add_argument("ARGS", nargs="*", help="Additional options to pass to the AI Model")


While you are in there?

Reply:
Done